src/main/resources/application-docker.yaml

```diff
@@ -0,0 +1,16 @@
+spring:
+  autoconfigure:
+    exclude: org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
+  data:
+    mongodb:
+      uri: mongodb://mongo:27017/telegram-bot-db
```

Comment:
will it work with your docker compose?

Reply:
Yes, it's working

Reply:
@ajax-shulepa-a checked yesterday on 1-2-1. Everything is ok here.

Why you doubt